test/integration/patients/worklist/worklist.js

```diff
@@ -267,6 +271,20 @@ context('worklist page', function() {
     cy
       .go('back');
 
+    cy
+      .get('@firstRow')
+      .contains('Test Patient')
+      .click();
+
+    cy
+      .url()
+      .should('contain', 'patient/dashboard/1')
+      .wait('@routePatient');
+
+    cy
+      .go('back')
+      .wait('@routeFlows');
+
```

Comment:
To add coverage for this: https://coveralls.io/builds/58331475/source?filename=src%2Fjs%2Fviews%2Fpatients%2Fworklist%2Fflow_views.js#L76.

Don't really understand how this line was missing coverage. But I'm guessing it was from when the default actions vs. flows toggle was changed for the worklist tests.